src/MongoDB/BulkWrite.c

```diff
@@ -102,27 +102,27 @@ static bool php_phongo_bulkwrite_opts_append_document(bson_t *opts, const char *
 	return true;
 } /* }}} */
 
-#define PHONGO_BULKWRITE_APPEND_BOOL(opt, value) \
-	if (!BSON_APPEND_BOOL(boptions, (opt), (value))) { \
+#define PHONGO_BULKWRITE_APPEND_BOOL(opt, value)                                                                 \
```

Comment:
We can turn this setting one ``Off`` as well, in which case it doesn't touch the ``\``. (Although doing that now, would be a bit annoying).

Reply:
I'm find with leaving it on. This looks ugly in GitHub because it uses 8-space tabs, but I imagine it'd be an improvement in our editors.